some docs don't need to be distributed so we will remove them, we will also remove
cf-promise.8 because it's an empty manpage, we will use cf-promises.8 instead
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,10 +1,10 @@
 
 SUBDIRS = pub src tests docs inputs
 
-EXTRA_DIST = acconfig.h COPYING ChangeLog INSTALL NEWS README LICENSE
+EXTRA_DIST = acconfig.h ChangeLog
 
 docdir = @docdir@
-doc_DATA = README ChangeLog INSTALL NEWS
+doc_DATA = ChangeLog
 
 #
 # Some basic clean ups
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -1,5 +1,5 @@
 # Info used in building the MAN files
-man_MANS = cf-agent.8 cf-serverd.8 cf-execd.8 cf-monitord.8 cf-report.8 cf-know.8 cf-runagent.8 cf-promise.8 cf-key.8
+man_MANS = cf-agent.8 cf-serverd.8 cf-execd.8 cf-monitord.8 cf-report.8 cf-know.8 cf-runagent.8 cf-promises.8 cf-key.8
 
 # Info used in building the Info files
 info_TEXINFOS = cf3-Reference.texinfo
